pkgs/games/steam/default.nix

```diff
@@ -19,6 +19,7 @@ let
         then pkgs.pkgsi686Linux.steamPackages.steam-runtime-wrapped
         else null;
     };
+    steamcmd = callPackage ./steamcmd.nix;
```

Comment:
I think we want to initialize it here with ```callPackage ./steamcmd.nix {}```. That way you should be able to get rid of the inherit steam thing below.

Reply:
:+1: